pkg/vm/context.go

```diff
@@ -354,3 +354,18 @@ func DynamicOnUnload(v *VM, ctx *Context, commit bool) error {
 func (c *Context) BreakPoints() []int {
 	return slices.Clone(c.sc.breakPoints)
 }
+
+// ArgumentsSlot returns the arguments slot of Context.
+func (c *Context) ArgumentsSlot() *Slot {
```

Comment:
The other option, btw, is to expose `Slot` with `Get()`, `Size()` and `MarshalJSON()` only (hide other methods). This way you'd get the `stackitem.Null{}` behavior of `Get()` which may be more interesting than checking for `nil` and `stackitem.Null{}`.

Reply:
> In that sense you can expose SlotArguments() []stackitem.Item

BTW, that was an original Erik's suggestion, it's just I messed all things up and suggested to expose the `Slot` itself, because this way we can implement some other useful methods on `Slot`.

> The other option, btw, is to expose Slot with Get(), Size() and MarshalJSON() only 

So this approach looks a bit more preferable to me, because it allows to provide some additional `Slot`-related functionality to the users and extend it in future if needed.

Reply:
Please discuss internally and give me a finalized choice of
1. expose as `[]stackitem.Item`
or 
3. expose as `*Slot` 
   Includes the following changes as per comments above
   - expose `refCounter` & `init()`
   - make `Set()` and `ClearRefs` private?

Reply:
Let's agree on 2.2 and also refactor `Size()` in order not to panic. `Dump*Slot` commands may be moved to CLI itself.